xla/service/gpu/stream_attribute_annotator.cc

```diff
@@ -130,26 +131,45 @@ absl::StatusOr<bool> StreamAttributeAnnotator::Run(
   XLA_VLOG_LINES(
       5, "StreamAttributeAnnotator::Run(), before:\n" + module->ToString());
   bool changed = false;
+  int64_t host_memory_color =
+      static_cast<int64_t>(stream_executor::MemoryType::kHost);
```

Comment:
looks like this value is fixed to be 5? I think that has a higher chance to have a clash with other instructions then it would be serialized. Maybe use `hlo_query::NextChannelId(*module)` for setting the stream id?

Reply:
Thanks. Will use NextChannelId() instead. AnnotateStreamAttributesForUsers() makes sense to propagate the channel id through the def-use chain.